libs/docs/platform/smart-filter-bar/examples/platform-smart-filter-bar-custom-filter-example.component.ts

```diff
@@ -101,7 +102,7 @@ export class PlatformSmartFilterBarDateRendererComponent extends BaseSmartFilter
     selector: 'fdp-platform-smart-filter-bar-custom-filter-example',
     templateUrl: './platform-smart-filter-bar-custom-filter-example.component.html',
     standalone: true,
-    providers: [provideDateTimeFormats()],
+    providers: [provideDateTimeFormats(), SmartFilterBarService, FormGeneratorService],
```

Comment:
I don't think users should provide this... Can this be done in SmartFilterBar component itself?

Reply:
It's already there, problem is that in this case user wants to add custom controls